e2e_test/streaming/aggregate/approx_percentile.slt

```diff
@@ -0,0 +1,102 @@
+# Single phase approx percentile
+statement ok
+create table t(p_col double, grp_col int);
+
+statement ok
+insert into t select generate_series, 1 from generate_series(-1000, 1000);
+
+statement ok
+flush;
+
+query I
+ select
+     approx_percentile(0.01, 0.01) within group (order by p_col) as p01,
+     approx_percentile(0.1, 0.01) within group (order by p_col) as p10,
+     approx_percentile(0.5, 0.01) within group (order by p_col) as p50,
+     approx_percentile(0.9, 0.01) within group (order by p_col) as p90,
+     approx_percentile(0.99, 0.01) within group (order by p_col) as p99
+ from t group by grp_col;
+----
+-982.5779489474152 -804.4614206837127 0 804.4614206837127 982.5779489474152
+
+query I
+select
+  percentile_disc(0.01) within group (order by p_col) as p01,
+  percentile_disc(0.1) within group (order by p_col) as p10,
+  percentile_disc(0.5) within group (order by p_col) as p50,
+  percentile_disc(0.9) within group (order by p_col) as p90,
+  percentile_disc(0.99) within group (order by p_col) as p99
+from t group by grp_col;
+----
+-980 -800 0 800 980
+
+statement ok
+create materialized view m1 as
+ select
+     approx_percentile(0.01, 0.01) within group (order by p_col) as p01,
+     approx_percentile(0.1, 0.01) within group (order by p_col) as p10,
+     approx_percentile(0.5, 0.01) within group (order by p_col) as p50,
+     approx_percentile(0.9, 0.01) within group (order by p_col) as p90,
+     approx_percentile(0.99, 0.01) within group (order by p_col) as p99
+ from t group by grp_col;
+
+query I
+select * from m1;
+----
+-982.5779489474152 -804.4614206837127 0 804.4614206837127 982.5779489474152
+
+# statement ok
+# recover;
+
+# wait recovery
+# sleep 10s
+
+query I
+select * from m1;
+----
+-982.5779489474152 -804.4614206837127 0 804.4614206837127 982.5779489474152
+
+# Test state encode / decode
+onlyif can-use-recover
```

Comment:
In which cases we will set label `can-use-recover`?

Reply:
Only for e2e tests.

Reply:
`recover` should not be used in cases where tests can execute in parallel.